DelvUI/Interface/HudManager.cs

```diff
@@ -31,6 +32,12 @@ public class HudManager
         private Dictionary<uint, JobHudTypes> _jobsMap;
         private Dictionary<uint, Type> _unsupportedJobsMap;
 
+        public HudHelper helper { get; } = new HudHelper();
```

Comment:
Any reason this is not private?
Also name should be Helper (if it remains public) or _helper (if its changed to private); to go in line with the current naming conventions in the code.

Reply:
sry, that used to be private property, i’ll have it renamed later today

Reply:
I’ll go with public since there is  at least  one case where i need to set flags  from Plugin.Draw